tests/diesel/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+#![cfg(test)]
+
+mod models;
+mod schema;
+
+use compact_str::format_compact;
+use diesel::prelude::*;
+use diesel::sql_query;
+use tempfile::tempdir;
+
+use crate::models::{
+    NewTodo,
+    Todo,
+};
+use crate::schema::todos;
+use crate::schema::todos::dsl::*;
+
+const TITLE: &str = "Say hello!";
```

Comment:
Could make it a `CompactString`:

```suggestion
const TITLE: CompactString = CompactString::from_static_str("Say hello!");
```

Kind of testing the static str impl and comparison with inline impl.

Reply:
Good idea!